grunt/tasks/translate/parseSchema.js

```diff
@@ -1,120 +1,112 @@
-          type: "models",
-          globPattern: "core/schema/*.schema",
-          bowerAttr: false,
-          schemaLabel: "models"
-        },{
-          type: "components",
-          globPattern: "components/*/properties.schema",
-          bowerAttr: "component",
-          schemaLabel: "components"
-        },{
-          type: "extensions",
-          globPattern: "extensions/*/properties.schema",
-          bowerAttr: "targetAttribute",
-          schemaLabel: "extensions"
-        },{
-          type: "menu",
-          globPattern: "menu/*/properties.schema",
-          bowerAttr: "targetAttribute",
-          schemaLabel: "menu"
-        }
-      ].forEach(function (item) {
+
+    function getSchemaData() {
+      [{
+        type: "models",
+        globPattern: "core/schema/*.schema",
+        bowerAttr: false,
+        schemaLabel: "models"
+      }, {
+        type: "components",
+        globPattern: "components/*/properties.schema",
+        bowerAttr: "component",
+        schemaLabel: "components"
+      }, {
+        type: "extensions",
+        globPattern: "extensions/*/properties.schema",
+        bowerAttr: "targetAttribute",
+        schemaLabel: "extensions"
+      }, {
+        type: "menu",
+        globPattern: "menu/*/properties.schema",
+        bowerAttr: "targetAttribute",
+        schemaLabel: "menu"
+      }].forEach(function(item) {
         global.translate.schemaData[item.schemaLabel] = {};
-        grunt.file.expand(srcPath+item.globPattern).forEach(function (filepath) {
+        grunt.file.expand(srcPath + item.globPattern).forEach(function(filepath) {
           var dir = path.parse(filepath).dir;
           var propertiesSchema = grunt.file.readJSON(filepath);
           var key;
           if (item.bowerAttr) {
-            var bower = grunt.file.readJSON(path.join(dir,"bower.json"));
+            var bower = grunt.file.readJSON(path.join(dir, "bower.json"));
             key = bower[item.bowerAttr];
           } else {
             key = path.parse(filepath).name.split(".")[0];
           }
-          
+
           global.translate.schemaData[item.schemaLabel][key] = {};
           global.translate.schemaData[item.schemaLabel][key].properties = propertiesSchema.properties || null;
           global.translate.schemaData[item.schemaLabel][key].globals = propertiesSchema.globals || null;
         });
       });
     }
-    
-    function processGlobals () {
-      [
-        {
-          type: "components",
-          schemaKey: "components", // key to find in global.translate.schemaData
-          schemaLabel: "_components" // name used to save in global.translate.schemaData.models.course
-        },{
-          type: "extensions",
-          schemaKey: "extensions",
-          schemaLabel: "_extensions"
-        },{
-          type: "menu",
-          schemaKey: "menu",
-          schemaLabel: "_menu"
-        }
-      ].forEach(function (item) {
+
+    function processGlobals() {
+      [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?